run.py

```diff
@@ -65,5 +67,16 @@ def run_streams():
         'server.thread_pool': 8
     }
 }
+
+if args.overlay:
+    overlay_dir = os.path.realpath(os.path.join(mydir, 'livestream-overlay'))
+    config['/livestream-overlay'] = {
+        'tools.staticdir.on': True,
+        'tools.staticdir.dir': overlay_dir,
+        'tools.caching.on': False,
+        'tools.staticdir.index': 'stream.html',
+    }
+    print('Livestream overlay is available at: http://localhost:5112/livestream-overlay/')
```

Comment:
Is this print necessary? The user is going to know that they've passed the `--overlay` option.
If we do need this, please could we ensure that the url can't get out of step with the actual config?

Reply:
In hindsight, no. I've moved this information to the readme, alongside the other path information in 
6096ee9.